src/chapters/inheritance/exercises.rst

```diff
@@ -0,0 +1,11 @@
+Exercises: Inheritance
+======================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project.
```

Comment:
I'd probably direct them to the technology package, otherwise it could be hard to find the right spot